resources/views/auth/register/personal.blade.php

```diff
@@ -21,6 +21,16 @@
         @error('date_of_birth')
         <span class="helper-text" data-error="{{ $message }}"></span>
         @enderror
+        <script>
```

Comment:
Do not forget to remove file from the mix compiler as well (webpack.mix.js). The build is failing now.